packages/chat/src/components/LinkPreview/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { forwardRef, type PropsWithRef } from 'react';
+
+import Icon from '../Icon';
+import { LoadingSpinner } from '../LoadingSpinner/LoadingSpinner';
+import {
+  linkPreviewContainer,
+  linkPreviewDetails,
+  linkPreviewImage,
+  linkPreviewImageContainer,
+  linkPreviewTitle,
+  linkPreviewUrl,
+} from './styles.css';
+import { getImageTypeFromUrl } from './utils';
+
+interface ILinkPreviewProps {
+  image?: string;
+  title: string;
+  link: string;
+  inline?: boolean;
+}
+
+const LinkPreview = forwardRef<HTMLDivElement, PropsWithRef<ILinkPreviewProps>>((props: any, ref) => {
+  const { image, title, link, inline } = props;
+  const linkType = getImageTypeFromUrl(link);
+  const shadow = linkType === 'custom' && !image;
+
+  return (
+    <div ref={ref} style={{ width: '256px' }} className={linkPreviewContainer({ inline })} {...props}>
```

Comment:
why inline styles?

Reply:
you're right! my bad, this was temporary to see it as if it was inside a proper Message component.. 
i'll make a note to fix this